lib/ansible/modules/cloud/amazon/execute_lambda.py

```diff
@@ -262,7 +262,7 @@ def main():
 
     if invoke_params['InvocationType'] == 'RequestResponse':
         try:
-            results['output'] = json.loads(response['Payload'].read())
+            results['output'] = json.loads(response['Payload'].read().decode('utf8'))
```

Comment:
@ryansb  both utf-8 and utf8 are legit aliases for the python utf_8 encoding (https://docs.python.org/2/library/codecs.html - section "7.8.3. Standard Encodings").  I personally don't see any difference; is there some Ansible standard?

@sivel what would be the correct usage here?  to_bytes(json.loads(response['Payload'].read()) ?